cmd/hashstate.cpp

```diff
@@ -157,31 +159,31 @@ uint64_t extract_incarnation(ByteView encoded) {
         uint8_t len = encoded[pos++];
         auto [incarnation, err]{rlp::read_uint64(encoded.substr(pos, len))};
         if (err != rlp::DecodingResult::kOk) {
-            throw err; 
+            throw std::runtime_error("Decoding error " + std::string(magic_enum::enum_name(err)));
```

Comment:
Perhaps we can create a dedicated `DecodingError` exception.

Reply:
I was thinking the same thing but realized rlp decoding is part of core.
Need to check if magic_enum builds properly for wasm